# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/navigation-timing/supported_navigation_type.window.js.ini
# Commit: 3cd097a7b017
# Full Hash: 3cd097a7b017154cf5f7a2d9ed97c7891d3074f4
# Author: Will Hawkins <whawkins@mozilla.com>
# Date: 2019-04-10 09:58:56
# Regressor Bug: 1539006
# File Overlap Count: 1
# Description:
#   Bug 1539006: Properly implement Performance Timeline Level 2 w3c spec. r=baku,flod
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D26654
# ==============================================================================

diff -r ea2baf2c9ae7 -r 3cd097a7b017 testing/web-platform/meta/navigation-timing/supported_navigation_type.window.js.ini
--- a/testing/web-platform/meta/navigation-timing/supported_navigation_type.window.js.ini	Tue Apr 09 20:32:54 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,4 +0,0 @@
-[supported_navigation_type.window.html]
-  [supportedEntryTypes contains 'navigation'.]
-    expected: FAIL
-